### PR TITLE
More work on flake8

### PR DESCRIPTION
**Pull request by tehw0lf**
Hello there, I have taken the liberty to combat some more flake8 errors :)

### CODE REVIEW

File: src/pybel/struct/graph.py [hunk 1]
```diff
@@ -250,8 +250,7 @@ def defined_namespace_keywords(self):
         :rtype: set[str]
         """
         return (
-                set(self.namespace_pattern) |
```
Comment (cthoyt) [inline src/pybel/struct/graph.py]: I disagree with this change (even if flake8 says so). It's much harder to read

File: src/pybel/__init__.py [hunk 2]
```diff
@@ -83,7 +83,7 @@
 PyBEL will build a new database and populate it on the next run.
 """
 
-from . import canonicalize, constants, examples, examples, io, struct
```
Comment (cthoyt) [inline src/pybel/__init__.py]: nice catch

File: src/pybel/struct/__init__.py [hunk 3]
```diff
@@ -21,11 +21,11 @@
 from .summary import *
 
 __all__ = (
-        graph.__all__ +
```
Comment (cthoyt) [inline src/pybel/struct/__init__.py]: huh. for some reason, pycharm adds this extra tab. At some point, I finally gave up trying to fight it @scolby33 what do you think?

File: tests/test_parse/test_parse_bel_relations.py [hunk 4]
```diff
@@ -487,7 +487,9 @@ def test_nested_failure(self):
             self.parser.relation.parseString(statement)
 
     def test_nested_lenient(self):
-        """ 3.1 \ Test nested statement"""
```
Comment (cthoyt) [inline tests/test_parse/test_parse_bel_relations.py]: This is actually getting worse for flake8. Should read 
```
"""Test nested statement (3.1)."""
```

File: tests/test_parse/test_parse_bel_relations.py [hunk 5]
```diff
@@ -910,7 +912,7 @@ def test_has_reaction_component(self):
         statement = 'rxn(reactants(a(CHEBI:"(S)-3-hydroxy-3-methylglutaryl-CoA"),a(CHEBI:NADPH), \
                     a(CHEBI:hydron)),products(a(CHEBI:mevalonate), a(CHEBI:"CoA-SH"), a(CHEBI:"NADP(+)"))) \
                     hasReactant a(CHEBI:"(S)-3-hydroxy-3-methylglutaryl-CoA")'
-        result = self.parser.relation.parseString(statement)
```
Comment (cthoyt) [inline tests/test_parse/test_parse_bel_relations.py]: I think this test is just checking that there's no exception thrown. The assignment can be removed, but the expression should stay

File: tests/test_struct/test_struct_graph.py [hunk 6]
```diff
@@ -201,7 +201,7 @@ def test_add_node_with_variant(self):
         graph = BELGraph()
         namespace, name, identifier, variant_name = n(), n(), n(), n()
         node = protein(namespace=namespace, name=name, identifier=identifier, variants=hgvs(variant_name))
-        parent = node.get_parent()
```
Comment (cthoyt) [inline tests/test_struct/test_struct_graph.py]: Same here - I think the point of this is just to test that the function doesn't throw an error. We could also check that it's not none and an instance of `pybel.dsl.BaseEntity`.

**Comment by cthoyt** [comment]
@tehw0lf Thanks for the PR! I made a couple comments - just a couple easy fixes and then I'll merge :smile:

Just out of curiosit - are you using PyBEL, and if so, for what?

File: src/pybel/struct/__init__.py (hunk 3 above)
Comment (scolby33) [reply src/pybel/struct/__init__.py]: I'm afraid I don't have an answer for you. What does Black do? Is it worth it to try to change the pycharm configuration? Either way *looks* fine to me.

**Comment by tehw0lf** [comment]
@cthoyt You're welcome!
I will see to fixing the remaining stuff soon.

Actually I am not using PyBEL, I was trying to find open flake8 issues to help and to progress in digital ocean's hacktoberfest :)

**Comment by tehw0lf** [comment]
Oops, there's a typo in the commit message.

**Comment by cthoyt** [comment]
Everything's looking good. Thanks so much for getting involved @tehw0lf. 

I saw on your GitHub profile you're in Dortmund/Bochum, so next time I visit I'll have to buy you a beer :)

**Comment by tehw0lf** [comment]
@cthoyt you're welcome! And yes indeed, I'm living (and working) in Dortmund and will not decline this offer :)

**Pull request merged**